Annotato/Annotato/Views/Components/Extensions/UIView.swift

```diff
@@ -45,4 +45,26 @@ extension UIView {
         button.setImage(UIImage(systemName: imageName), for: .normal)
         return button
     }
+
+    class func makeSaveButton() -> UIButton {
+        makeButtonWithString(label: "Save")
+    }
+
+    class func makeDiscardButton() -> UIButton {
+        makeButtonWithString(label: "Discard")
+    }
+
+    class func makeButtonWithString(label: String) -> UIButton {
+        let button = UIButton()
+        button.setTitle(label, for: .normal)
+        button.setTitleColor(UIColor.blue, for: .normal)
+        return button
+    }
```

Comment:
```suggestion
    class func makeTextButton(label: String) -> UIButton {
        let button = UIButton()
        button.setTitle(label, for: .normal)
        button.setTitleColor(UIColor.blue, for: .normal)
        return button
    }
```

This can stay here since it can be reused by other views.

Reply:
Ok can